pavement.py

```diff
@@ -6,7 +6,7 @@
 NAME = 'blackbelt'
 VERSION = '0.12.1'
 
-requires = ['click', 'requests', 'PyGithub', 'slacker', 'pyopenssl', 'ndg-httpsclient', 'pyasn1']
+requires = ['click', 'requests', 'PyGithub', 'slacker==0.6.2', 'pyopenssl', 'ndg-httpsclient', 'pyasn1']
```

Comment:
@abtris It should be on both places. 

I _think_ @honzajavorek told me that we actually can get rid of `requirements.txt`, so ask him.